cosmic/tests/data/Params.ini

```diff
@@ -13,14 +9,55 @@ select_final_state = True
 ; default=[0,1,2]
 binary_state = [0,1,2]
 
-; lisa_sources decides whether to retain only sources within LISA's frequency range (porb<1e5 sec)
-; default=False
-lisa_sources = False
+[sampling]
+; Specify if you woud like to sample initial conditions via
+; the independent method (indepedent) or would like to sample
+; initial conditions follow Moe & Di Stefano (2017) (multidim)
+initial_samp=multidim
```

Comment:
Same as my comment above, we should think about changing this to 'samp_method' or something